float64 xmin
float64 xmax
float64 ymin
float64 ymax
uint16[] scene_ids
float64[] confidences
string[] labels


